style/components/comments.css

```diff
@@ -0,0 +1,55 @@
+.comments {
+  display: flex;
+  flex-direction: column;
+  gap: 1.5rem;
+}
+
+.comments .textarea {
+  border: 1px solid var(--color-border);
+  border-radius: var(--border-radius);
+  padding: 0.55rem;
+  transition: border-color var(--transition-duration);
+}
+
+.comments .textarea:focus-within {
+  border-color: var(--color-primary);
+}
+
+.comments .textarea textarea {
+  border: none;
+  resize: none;
+  padding: 0 0.4rem;
+}
+
+.comments .textarea .actions {
+  display: flex;
+}
+
+.comments .textarea .actions .submit {
+  margin-right: auto;
+}
+
+.comments .attachments,
+.comments .comment {
+  display: flex;
+  gap: 1rem;
+}
+
+.comments .comment .text-info > * {
+  margin-bottom: 0.5rem;
+}
+
+.comments .attachments {
+  align-items: center;
+}
+
+.comments .comment .author {
+  color: var(--color-heading);
+  margin-bottom: 0.5rem;
+  display: inline-block;
+  margin-right: 0.5rem;
```

Comment:
So you'll give this margin to all elements that you theoretically will have to add later to this row?